libs/ledger-live-common/src/bridge/react/SyncOneAccountOnMount.ts

```diff
@@ -3,17 +3,20 @@ import { useBridgeSync } from "./context";
 export const SyncOneAccountOnMount = ({
   priority,
   accountId,
+  reason = "one-account",
 }: {
   accountId: string;
   priority: number;
+  reason: string;
```

Comment:
Should `reason` by typed with an enum? As a way of documenting the possible values if we want a strict list, or do we want the reason to be a free field?

Reply:
free field is ok, there will always be more reason and it's out of control from live-common perspective, most of the reasons are defined by the user-land side (LLD, LLM)